freqtrade/freqai/data_kitchen.py

```diff
@@ -450,6 +454,31 @@ def split_timerange(
         # print(tr_training_list, tr_backtesting_list)
         return tr_training_list_timerange, tr_backtesting_list_timerange
 
+    def split_timerange_live_models(
+        self
+    ) -> Tuple[list, list]:
+
+        tr_backtesting_list_timerange = []
+        pair = self.pair.split("/")[0].split(":")[0]
+        if pair not in self.backtest_live_models_data["pairs_end_dates"]:
+            raise OperationalException(
+                f"Model not available for pair {self.pair}. "
+                "Please, try again after removing this pair from the configuration file."
+            )
+        pair_data = self.backtest_live_models_data["pairs_end_dates"][pair]
+        model_end_dates = []
+        backtesting_timerange = self.backtest_live_models_data["backtesting_timerange"]
+        for end_date in pair_data:
+            model_end_dates.append(end_date)
```

Comment:
List-comprehension would yield better performance ... ;) 

you'll still need the final append - but still ...